lib/modules/apostrophe-doc-type-manager/public/js/chooser.js

```diff
@@ -324,6 +325,50 @@ apos.define('apostrophe-doc-type-manager-chooser', {
         position: { collision: 'flip' }
       });
     };
+
+    self.enableSortable = function () {
+      var sortableOptions = {
+        handle: '[data-apos-sort-item]',
+        update: function(event, ui) {
+          var $choices = self.$el.find('[data-chooser-choice]');
+          var proposedChoices = [];
```

Comment:
This would be a better place for a quick sanity check: if not $choices.length then we can't do this and should just cancel.

Reply:
just checking the length of the DOM now